app/controllers/records_controller.rb

```diff
@@ -27,10 +27,14 @@ def destroy
     head :no_content
   end
 
+  def update_records
+    @record_update = Record.find(params[:id])
+    @record_update.update(schedule_date: params[:schedule_date])
+  end
```

Comment:
Let's also update nullable `specialist_id`. This is where a specialist claims a user record. We need to update both `shedule_date` and `specialist_id`

Reply:
Let me work on it asap. But I assume the route looks okay?? that is where i had my doubts
@wathigo

Reply:
@wathigo check it out now

Reply:
Let mee see